massgov/mayflower-patternlab
# DP 7341  add about pattern for elected officials

## Description
Adds a person card organism and an About section organism for use on elected officials pages

## Related Issue / Ticket

- [JIRA issue](https://jira.state.ma.us/browse/DP-7341)

## Steps to Test
1. View the [circular image atom](https://clair0917.github.io/mayflower/?p=atoms-image-circular) here
2. View the [person card organism](https://clair0917.github.io/mayflower/?p=organisms-person-card) here
3. View [the organization elected official page](https://clair0917.github.io/mayflower/?p=pages-organization-elected-official) to see about section in place


## Screenshots
![about-section](https://user-images.githubusercontent.com/18662734/37412254-f2b3b676-277a-11e8-88d4-c3166ee4924e.jpg)

## Additional Notes:

Anything else to add?

#### Impacted Areas in Application
* Image Atom
* Organization Elected Officials Page
